docs/github.md

```diff
@@ -0,0 +1,41 @@
+# Using Jujutsu with Github Projects
+
+## Setup a ssh key
+As of December 2022 it's necessary to setup a ssh key to work with github
+projects. See [Github's Tutorial][gh]. This restriction may be lifted in the 
+future, see issue #469 for more information and progress on authenticated http. 
+
+
+## Basic workflows
+Jujutsu supports both a `git` and a `hg` like workflows. This allows you
```

Comment:
I think it's better to describe the workflows you have in mind and then maybe compare specifics to Git and Mercurial. Something like this:

> You can either do X and the to do Y (similar to how Git is typically used),
> or you can do Z (similar to how Mercurial is typically used)

That way it's clearer to the reader what you mean by Git-like and Mercurial-like.